### DISCUSSION TITLE
Transient simulation in density solver not available

### DISCUSSION THREAD

**Discussion by jaydev-dave** [nextfoam/baram#128] [Q&A]
As tile say, why transient option is not availabe in density based solver? 
Are there any plan to activate it?

**Comment by kdg5424** [comment]
Hello @jaydevdave97. This is Donggyu from NEXTFOAM.
Unfortunately, we've not develop the solve. (Density-based solver with transient option)
We have the plan for it but not sure when it will be.
If there is any fund, we could develop it faster.

Thanks for your interest

Best regards

Donggyu Kim
